cmd/registry/user.go

```diff
@@ -73,6 +74,35 @@ var userGetCmd = &cobra.Command{
 	},
 }
 
+var usersReverseLookupCmd = &cobra.Command{
+	Use:   "reverse",
```

Comment:
instead of `kld registry get reverse` can I recommend `kld registry get userByOrg`

Reply:
The new `usersReverseLookupCmd` takes an Ethereum account address as an input and returns the human-readable string that is the owner of that account. The owner could be an individual user's name (i.e. `sam.stokes@consensys.net`) or it could be an organization (i.e. `kaleido.io`). To me `userByOrg` would indicate I need to input an organization name to return a username. Maybe `kld registry get userByAccount` or `kld registry get userReverse` would be more appropriate?

Reply:
Decided to go with `userByAccount`